src/components/tasks/TasksContent.tsx

```diff
@@ -109,18 +111,38 @@ export const TasksContent = ({ dev }: { dev: boolean }) => {
     if (isLoading) return <p>Loading...</p>;
 
     if (isError) return <p>{TASKS_FETCH_ERROR_MESSAGE}</p>;
+    const taskSelectOptions = TABS.map((item) => ({
+        label: changeName(item),
+        value: item,
+    }));
 
     return (
         <div className={classNames.tasksContainer}>
-            <TabSection onSelect={onSelect} activeTab={selectedTab} />
+            <div className={classNames['status-tabs-container']}>
+                <TabSection onSelect={onSelect} activeTab={selectedTab} />
+            </div>
+            <div className={classNames['status-select-container']}>
+                <Select
+                    value={{
+                        label: changeName(selectedTab),
+                        value: selectedTab,
+                    }}
+                    onChange={(o) => {
+                        if (o) {
+                            console.log(o);
+                            onSelect(o.value as Tab);
+                        }
+                    }}
```

Comment:
can we name this something more meaningful other than `o`
also please remove the console.log

Reply:
resolved